youtube_dl/extractor/olympicchannel.py

```diff
@@ -0,0 +1,55 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import ExtractorError, try_get, unified_strdate
+
+
+class OlympicChannelIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?olympicchannel\.com/../playback/(?P<id>.+)'
```

Comment:
Should only capture path part and should not capture query or fragment parts of URL.

Reply:
Done